istio/release/samples/bookinfo/kube/route-rule-all-v1.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: config.istio.io/v1alpha2
 kind: RouteRule
 metadata:
   name: productpage-default
+  namespace: default
```

Comment:
revert these changes.   the rule file should not be changed

Reply:
ok